src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -113,12 +109,6 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
     }
   }
 
-  protected def sourceRelations(spark: SparkSession, df: DataFrame): Seq[Relation] =
```

Comment:
This is no longer needed as creating `Relation` has moved to `FileBasedRelation`.